### ISSUE TITLE
Something about the training accuracy

### ISSUE THREAD

**Issue by LiJianGuo2333** [roomylee/rcnn-text-classification#10]
Thanks for your code ! I wanna ask what is your accuracy on the training set and dev set during the training process ? I am a little bit confused because my best result is 0.67 for the training set, 0.55 for the dev set. I do not use pre-training _word2vec_, _num_epochs_ was set to 50, and other experiment configurations are just default
